### ISSUE TITLE
project face generation - weight init documentation

### ISSUE THREAD

**Issue by aravind-deva** [udacity/deep-learning-v2-pytorch#321]
Hello
in the project face generation - weight initialization exercise, it is suggested in 'Notes' that only conv and linear layers have to be inited. But in the code comments 'BatchNorm2d' is also included.(and this is suggested in review as well)

Exercise: Complete the weight initialization function
This should initialize only convolutional and linear layers

 # classname will be something like:
    # `Conv`, `BatchNorm2d`, `Linear`, etc.

**Comment by abhiojha8** [comment]
Adding batch normalization layer might certainly help that is why it is added in comments. For more clarification please reach out to our mentors on Knowledge Hub.
